src/idpconfgen/cli_bgeo.py

```diff
@@ -126,11 +136,51 @@ def main(pdb_files, convert=False, func=None):
     if convert:
         log.info(S('Converting'))
         converted = convert_bond_geo_lib(bond_geo_db)
+        log.info(S('done'))
         save_dict_to_json(converted, output='bgeo.json')
-
     else:
         save_dict_to_json(bond_geo_db, output='bgeo.json')
-
+    
+    if plot:
+        log.info(S('Plotting bond angle distributions in radians...'))
+        
+        #avoid hardcoding
+        bend_names = []
+        first = list(bond_geo_db.items())[0][1]
+
+        styled_names = {
+            "Cm1_N_Ca": r"$C_{-1}-N-C_{\alpha}$",
+            "N_Ca_C": r"$N-C_{\alpha}-C$",
+            "Ca_C_Np1": r"$C_{\alpha}-C-N_{+1}$",
+            "Ca_C_O": r"$C_{\alpha}-C-O$",
+            }
+
+        for key in first:
+            bend_names.append(styled_names.get(key, key))
+        
+        bend_angles = np.ndarray(shape=(len(bond_geo_db), len(bend_names)))
```

Comment:
I suggest adding an option to convert to degrees also. It can go to `plotvars`, be handled by `cli_bgeo` and doesn't need to be passed to the plot function itself.